### PR TITLE
WIP: Testing environments, do not merge

### PR DESCRIPTION
**Pull request by Lorquas**
This is to test the environment for https://github.com/candlepin/candlepin/pull/3001

### CODE REVIEW

File: jenkins/Jenkinsfile [hunk 1]
```diff
@@ -106,17 +106,17 @@ pipeline {
                         checkout scm
                         sh 'docker login -u "$CANDLEPIN_QUAY_BOT_USR" -p "$CANDLEPIN_QUAY_BOT_PSW" quay.io'
                         sh 'python jenkins/check_pr_branch.py $CHANGE_ID'
-                        sh 'docker logout quay.io'
+                        // sh 'docker logout quay.io'
```
Comment (nikosmoum) [inline jenkins/Jenkinsfile:109]: @Lorquas Is commenting this out something we should do? Do you think it is responsible for the 'unauthorized' errors we sometimes see when trying to pull images?

**Comment by nikosmoum** [comment]
@Lorquas I tried re-running the pipeline, but now it's stuck and says `There are no nodes with the label ‘jmolet-vm0’`

**Comment by nikosmoum** [comment]
@Lorquas Could you setup the `jmolet-vm0` nodes again so we can investigate the spec test failures with your docker secret changes?

File: jenkins/Jenkinsfile (hunk 1 above)
Comment (Lorquas) [reply jenkins/Jenkinsfile:109]: @nikosmoum   yes i think this logout/login race condition is causing these errors, one test will log out right before another test needs to perform a pull.  I don't see the harm in just removing this line, i think you can just continuously log in over and over to refresh the credential.

Comment (sonalidhome) [reply jenkins/Jenkinsfile:109]: @Lorquas Logging out changes are already merged into the master branch (PR https://github.com/candlepin/candlepin/pull/3070). 
On rebasing this PR, you will no longer see the logout code.

**Comment by Lorquas** [comment]
Closing this PR, work will continue in a PR that will have mergeable changes.

**Pull request closed without merging**